emukit/examples/preferential_batch_bayesian_optimization/pbbo/inferences/sexpgp_comparison.stan

```diff
@@ -0,0 +1,56 @@
+functions {
```

Comment:
Can you briefly explain why this bit has to be implemented in Stan? not saying this is a no-go, just trying to understand why we suddenly need it in otherwise Python-only codebase

Reply:
It is a baseline method we compare our method against. Implementing the method on Stan is fast and the code is much more readable than making an implementation from scratch.

Reply:
ok, fair enough. in that case can you add a comment at the top of this file with the same explanation?